app/assets/stylesheets/themes/deathinspace.css

```diff
@@ -0,0 +1,35 @@
+:root {
+  --dis-black: 0deg 0% 0%;
+  --dis-red: 0deg 100% 30%;
+  --dis-white: 0deg 0% 100%;
+  
+  /* colors.accent */
+  --color--accent-1: var(--dis-green);
```

Comment:
Looks like this variable is missing and it makes the highlighted words in the search results appear as black text on a black background

<img width="444" alt="Screenshot 2022-10-29 at 17 26 32" src="https://user-images.githubusercontent.com/1191840/198842520-a240ccaf-16c0-4c0c-92e5-9cfa8e98f5e5.png">

Reply:
Oops, thanks for the catch!

Reply:
Fixed.